### DISCUSSION TITLE
Get the detail logs

### DISCUSSION THREAD

**Discussion by richman2024** [idaholab/moose#25885] [Q&A General]
How can I get the detail logs which are printed on the terminal?
Such as below information showed on the terminal after a simulation finished:
0 Nonlinear |R| = 9.214686e-01
      0 Linear |R| = 9.214686e-01
      1 Linear |R| = 2.506059e-01
      2 Linear |R| = 3.767411e-02
      3 Linear |R| = 3.042510e-03
      4 Linear |R| = 2.674655e-04
      5 Linear |R| = 3.080687e-05
 1 Nonlinear |R| = 5.707467e-04
      0 Linear |R| = 5.707467e-04
      1 Linear |R| = 9.549090e-06
      2 Linear |R| = 1.710096e-06
      3 Linear |R| = 5.100882e-07
      4 Linear |R| = 4.510555e-07
      5 Linear |R| = 3.378032e-07
      6 Linear |R| = 2.988210e-07
.....................
Linear solve did not converge due to DIVERGED_ITS iterations 50
 4 Nonlinear |R| = 4.671953e-09
 Solve Converged!
Currently Executing
  Finished Solving

**Comment by GiudGiud** [comment]
Hello

You can create a Console block in the [Outputs] section and specify the output_file parameter
see https://mooseframework.inl.gov/source/outputs/Console.html

Guillaume

**Reply by richman2024** [reply]
That's what I find, thank you!
